### ISSUE TITLE
Translations of new line characters is broken with hmr

### ISSUE THREAD

**Issue by tomitrescak** [gadicc/meteor-hmr#101]
Hi, Using the package the strings are not transpiled correctly, what makes string generation not possible. With ecmascript hot following string:

```javascript
return res.title + "\n" + res.text + "\n---------------------";
```

is transpiled to

```javascript
return res.title + "\
           " + res.text + "\
           "---------------------";
```

changing back to ecmascript package things are back to normal.

**Comment by clayne11** [comment]
Have you tried using template strings?

```javascript
return `${res.title}\n${res.text}\n---------------------`;
```

Not to say that the package shouldn't work with string concatenation, but it's probably better to just use template strings. Manually concatenating strings is discouraged in ES6.

**Comment by tomitrescak** [comment]
I did, same result :/

**Comment by tomitrescak** [comment]
Fun fact ... the result for both is the same ...

**Comment by clayne11** [comment]
Interesting. Good find.

**Comment by gadicc** [comment]
That's pretty weird.  There's nothing in meteor-hmr that could do this though.  I think it's a babel thing.  There have been quite a lot of updates to the meteor-babel preset lately (including just bumping dependency versions).  Could you test against Meteor `1.3.3-beta.1` with the same babelrc (but *without* the `meteor` preset which is auto-included at the right version), and see if you get the same behaviour?

**Comment by tomitrescak** [comment]
What is the babelrc that I should use? Thanks

**Comment by clayne11** [comment]
If you're using Meteor `1.3.3-beta.1` use whatever your regular `babelrc` is just without the `meteor` preset.

**Comment by gadicc** [comment]
@tomitrescak did this help resolve the issue?

I actually thought of one place where this could still be our fault :D  Does it work the *first* time with `ecmascript-hot` (on initial build and first page load) and only break on *hotloading* ?

**Comment by tomitrescak** [comment]
I'll have more time tomorrow, will get back to you. Did not have a chance to play with 1.3.3 yet.

**Comment by gadicc** [comment]
Ok great, thanks!  I heard some people had issues with `1.3.3-rc.0` (I haven't had a chance to check it out yet) so in case it doesn't work with that you can try `1.3.3-beta.2`.

**Comment by gadicc** [comment]
Oops, didn't mean to close this.  But I think it's fixed :D  Please confirm.  Published in `gadicc:ecmascript@2.0.0-beta.5`.
